autogpts/autogpt/autogpt/url_utils/validators.py

```diff
@@ -2,10 +2,11 @@
 import re
 from typing import Any, Callable, ParamSpec, TypeVar
 from urllib.parse import urljoin, urlparse
+from autogpt.config import Config
 
 P = ParamSpec("P")
 T = TypeVar("T")
-
+config = Config
```

Comment:
this won't work

Reply:
@Pwuts can you please help with why this wont work?

Reply:
from autogpt.config import ConfigBuilder
config = ConfigBuilder.build_config_from_env()

web_policy = config.web_policy
url_list = config.url_list

will this work? @Pwuts